# Representation of bus stops used for bus central to update sml_world_central

int32[] bus_stops
int32[] bus_stop_demands
int32[] closed_bus_stops
